dtmi/stmicroelectronics/steval_proteus1/standard_if-1.json

```diff
@@ -0,0 +1,301 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:stmicroelectronics:steval_proteus1:standard_if;1",
+  "@type": "Interface",
+  "description": {
+    "en": "Reports temperature, accelerometer, gyroscope, predictive speed status, predictive acceleration peak status, predictive frequency domain status, battery status"
+  },
+  "displayName": {
+    "en": "Standard interface for PROTEUS"
+  },
+  "contents": [
+    {
+      "@type": "Telemetry",
+      "comment": "Battery Level [%]",
+      "displayName": {
+        "en": "Battery [%]"
+      },
+      "name": "battery",
+      "schema": "double"
+    },
+    {
+      "@type": [ 
+        "Telemetry",
+        "Temperature"
+      ],
+      "comment": "Temperature in degrees Celsius from temperature sensor.",
+      "displayName": {
+        "en": "Temperature"
+      },
+      "name": "temperature",
+      "schema": "double",
+      "unit": "degreeCelsius"
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Acceleration X/Y/Z components [mg]",
+      "displayName": {
+        "en": "Acc Value [mg]"
+      },
+      "name": "accelerometer",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Single acceleration sample. I.e.: {a_x, a_y, a_z}."
```

Comment:
Per [DTDL Naming Guidelines](https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines), "acceleration" and "sample" should be capitalized.